tests/smoke-tests/pom.xml

```diff
@@ -1339,6 +1339,30 @@
             <sts-surefire-extra-args>--add-opens=java.management.rmi/javax.management.remote.rmi=ALL-UNNAMED --add-opens=java.rmi/sun.rmi.server=ALL-UNNAMED --add-opens=java.rmi/sun.rmi.transport=ALL-UNNAMED</sts-surefire-extra-args>
          </properties>
       </profile>
+
+      <profile>
+         <!-- Quick subset of tests, used on PR checks -->
+         <id>fast-tests</id>
+         <build>
+            <pluginManagement>
+               <plugins>
+                  <plugin>
+                     <groupId>org.apache.maven.plugins</groupId>
+                     <artifactId>maven-surefire-plugin</artifactId>
+                     <configuration>
+                        <includes combine.children="override">
+                           <include>**/smoke/upgradeTest/*Test.java</include>
+                        </includes>
+                     </configuration>
+                  </plugin>
+               </plugins>
+            </pluginManagement>
+         </build>
+         <properties>
+            <skipSmokeTests>false</skipSmokeTests>
+         </properties>
```

Comment:
I think this should be done in the root pom like it is for other modules participating in the fast-tests profile. Makes it easier to see at the root which modules tests are going to be enabled. It can reference that module additionally restricts which tests are run. 

E.g:
https://github.com/apache/activemq-artemis/blob/8078dd098ccc38cd57eefa3e94ba1dfe50c6865b/pom.xml#L1296-L1299